scripts/fvt-tools.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+# Wait loop configuration
+SLEEP_COUNT=30
+SLEEP_WAIT_SECONDS=2
+declare -a NODES
+
+fvtlog() {
+  echo "$(date +"%F %T %Z")" "[fvt]" "$*"
+}
+
+list_nodes() {
+  NODES=($(kubectl get nodes --selector "node-role.kubernetes.io/master notin ()" --no-headers -o jsonpath='{.items[*].metadata.name}'))
+}
+
+# Function to check the CR status
+# Parameters:
+# - CR name
+# - Node name (optional, needed when a CR applies only for a given node)
+check_staticroute_crd_status() {
+  set +e
+  cr=$1
+  match_node="${2:-all}"
+  status_ok=false
+  for _ in $(seq ${SLEEP_COUNT}); do
+    if [[ "${match_node}" == "all" ]]; then
+      cr_array=($(kubectl get staticroute "${cr}" --no-headers -o jsonpath='{.status.nodeStatus[*].hostname}'))
+      if [[ ${#NODES[*]} -eq ${#cr_array[*]} ]]; then
+          status_ok=true
+          break
+      fi
+    else
+      node=$(kubectl get staticroute "${cr}" --no-headers -o jsonpath='{.status.nodeStatus[*].hostname}' | grep "${match_node}")
+      if [[ "${node}" == "${node}" ]]; then
+          status_ok=true
+          break
+      fi
+    fi
+    sleep ${SLEEP_WAIT_SECONDS}
+  done
+  if [[ $status_ok == "false" ]]; then
```

Comment:
This expression make me confused. `$status_ok == false` or `"$status_ok" == "false"` would be more clear. I think first one is safe here, because we know the value of `status_ok`. `"$status_ok"` is useful when we are not sure about spaces in the string.